assets/tests/components/routeLadder.test.tsx

```diff
@@ -655,7 +655,62 @@ describe("routeLadder", () => {
     expect(tree).toMatchSnapshot()
   })
 
-  test("renders alert icon on a route ladder with an active detour", () => {
+  test("renders active detours in dropdown that were made in skate", async () => {
+    jest.mocked(getTestGroups).mockReturnValue([TestGroups.DetoursPilot])
```

Comment:
Do you need this test group for this test to work? IIRC everyone should be able to _open_ a detour from this entry point. 
(Same for other new tests that mock this test group.)